users/config.rst

```diff
@@ -180,12 +180,15 @@ autoNormalize
 The following child elements may exist:
 
 device
-    These must have the ``id`` attribute and nothing else. Mentioned devices
-    are those that will be sharing the folder in question. Each mentioned
-    device must have a separate ``device`` element later in the file. It is
-    customary that the local device ID is included in all repositories.
+    These must have the ``id`` attribute and can have a ``introducedBy`` attribute,
+    identifying the device that introduced us to share this folder with the given device.
+    If the original introducer unshares this folder with this device, our device will follow 
+    (subject to skipIntroductionRemovals being false on the introducer device) and unshare the folder.
```

Comment:
May be worth moving the parenthesised sentence to the end, as a fragment, to make this easier to read.